src/test/java/ru/ayaz/UserMessageHandlerTest.java

```diff
@@ -0,0 +1,127 @@
+package ru.ayaz;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+import static org.mockito.Mockito.*;
+
+public class UserMessageHandlerTest {
+
+    private UserMessageHandler messageHandler;
+
+    @Before
+    public final void before() {
+        int queueSize = 500;
```

Comment:
I think it is more readable as another developer willn't have to check what '500' parameter means switching to UserMessageHandler class file.